app/views/accounts/show.html.erb

```diff
@@ -1,22 +1,67 @@
+<% if @presenter.showing_alerts? %>
   <div class="margin-bottom-4">
     <% if @presenter.show_password_reset_partial? %>
       <%= render 'accounts/password_reset', presenter: @presenter %>
     <% end %>
-
-    <% if @presenter.show_gpo_partial? %>
-      <%= render 'accounts/pending_profile_gpo' %>
-    <% end %>
-
     <% if @presenter.show_service_provider_continue_partial? %>
       <%= render 'accounts/service_provider_continue', presenter: @presenter %>
     <% end %>
   </div>
 <% end %>
 
-<%= render 'accounts/header', presenter: @presenter %>
+<% if @presenter.showing_any_partials? %>
+  <div class="margin-bottom-4">
+    <div class="margin-bottom-4 card profile-info-box">
+      <div class="grid-row margin-bottom-1 margin-top-0">
+        <div class="grid-col-12">
+          <div class="grid-row flex-justify padding-y-2 tablet:padding-y-0 margin-bottom-2 bg-primary-lightest tablet:bg-transparent position-relative padding-bottom-4">
+            <div class="grid-col-12 tablet:grid-col-auto">
+              <h2 class="margin-0">
+                <div class="tablet:text-center">
+                  <%= t('account.index.verification.identity_verification') %>
+                </div>
+              </h2>
+            </div>
+            <div class="account-header__badges grid-col-12 tablet:grid-col-auto">
+              <%= render 'accounts/validated_badge' %>
+            </div>
+          </div>
+          <% unless @presenter.show_pii_partial? %>
+            <% if @presenter.show_verified_badge? %>
+              <%= render 'accounts/unverified_profile' %>
+            <% else %>
+              <p>
+                <%= t('account.index.verification.finish_verifying', partner_agency: @presenter.service_provider_or_app_name) %>
+                  <br><%= link_to t('account.index.verification.learn_more_link'), MarketingSite.help_url %>
+              </p>
+            <% end %>
+          <% end %>
+          <% if @presenter.show_gpo_partial? %>
+            <%= render 'accounts/pending_profile_gpo' %>
+          <% end %>
+          <% if @presenter.show_ipp_partial? %>
+            <%= render 'accounts/pending_profile_ipp' %>
+          <% end %>
+        </div>
+      </div>
+      
+      <% if @presenter.show_pii_partial? %>
+
+      <% if @presenter.biometric_identity_verification? %>
+        <p><%= t('account.index.verification.you_verified_your_biometric_identity', app_name: APP_NAME) %>
+          <%= link_to t('account.index.verification.learn_more_link'), MarketingSite.help_url %></p>
+      <% else %>
+        <p><%= t('account.index.verification.you_verified_your_identity', partner_agency: @presenter.service_provider_or_app_name) %>
+          <br><%= link_to t('account.index.verification.learn_more_link'), MarketingSite.help_url %></p>
+      <% end %>
+        <%= render 'accounts/pii', pii: @presenter.pii, locked_for_session: @presenter.locked_for_session %>
+      <% end %>
```

Comment:
let's add some indentation to make the matching if/else clearer
```suggestion
      <% if @presenter.show_pii_partial? %>
        <% if @presenter.biometric_identity_verification? %>
          <p><%= t('account.index.verification.you_verified_your_biometric_identity', app_name: APP_NAME) %>
            <%= link_to t('account.index.verification.learn_more_link'), MarketingSite.help_url %></p>
        <% else %>
          <p><%= t('account.index.verification.you_verified_your_identity', partner_agency: @presenter.service_provider_or_app_name) %>
            <br><%= link_to t('account.index.verification.learn_more_link'), MarketingSite.help_url %></p>
        <% end %>
        <%= render 'accounts/pii', pii: @presenter.pii, locked_for_session: @presenter.locked_for_session %>
      <% end %>
```

Reply:
I have adjusted that tabbing.